ext_mongodb.php

```diff
@@ -1,6 +1,12 @@
 <?hh
 namespace MongoDB\Driver;
 
+interface TypeWrapper
+{
+	function fromType(\MongoDB\BSON\Type $type);
+	function toType() : \MongoDB\BSON\Type;
```

Comment:
I realize the visibility is implied for interfaces, but we should be consistent about including it (or not). I'd suggest adding `public` to all of these.